networking_nsxv3/tests/unit/test_nsxv3_integration.py

```diff
@@ -40,7 +32,7 @@ def setUp(self):
 
     def test_create_logical_switch(self):
         sw_name = self.uuid
-        tz_id = NetworkTest.TRANSPORT_ZONE_ID
+        NetworkTest.TRANSPORT_ZONE_ID
```

Comment:
Is there any use in keeping this line? It looks like a NOOP to me, as `TRANSPORT_ZONE_ID` doesn't seem to be a `property()`.